src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6469,17 +6469,17 @@ private StatementSyntax ParseStatementNoDeclaration(bool allowAnyExpression)
                 case SyntaxKind.IdentifierToken:
                     if (isPossibleAwaitForEach())
                     {
-                        return this.ParseForEachStatement(parseAwaitKeywordForAsyncStreams());
+                        return this.ParseForEachStatement(parseAwaitKeyword(MessageID.IDS_FeatureAsyncStreams));
                     }
                     else if (isPossibleAwaitUsing())
                     {
                         if (PeekToken(2).Kind == SyntaxKind.OpenParenToken)
                         {
-                            return this.ParseUsingStatement(parseAwaitKeywordForAsyncStreams());
+                            return this.ParseUsingStatement(parseAwaitKeyword(MessageID.IDS_FeatureAsyncUsing));
                         }
                         else
                         {
-                            return this.ParseLocalDeclarationStatement(parseAwaitKeywordForAsyncStreams());
+                            return this.ParseLocalDeclarationStatement(parseAwaitKeyword());
```

Comment:
is this missing a possible LangVersion diagnostic for `using var`? Or is it reported elsewhere?

Reply:
No, previously we were reporting for both the `await` and the `using`. This change removes the `await` warning in this path so just the using gets the lang ver warning.